src/main/java/cz/mzk/fedora/model/DataStreams.java

```diff
@@ -4,7 +4,7 @@
 
 public enum DataStreams {
 
-    RELS_EXT("RELS-EXT", "application/rdf+xml" , "true", "X", "A");
+    RELS_EXT("RELS-EXT", "text/xml" , "true", "X", "A");
```

Comment:
proc "text/xml"?

Reply:
proc to tady potrebujes menit kdyz resis Solr?

Reply:
Toto som menil este pred spustenim skriptu na fedoru, lebo pozeral som si predosle verzie a tie to mali nastavene... a zabudol som to pred tym commitnut